File: library/src/main/java/com/getstream/sdk/chat/rest/response/SearchMessagesResponse.java
package com.getstream.sdk.chat.rest.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/*
 * Created by Anton Bevza on 2019-10-17.
 */
public class SearchMessagesResponse {

    @SerializedName("results")
    @Expose
    private List<MessageResponse> results;

    public List<MessageResponse> getResults() {
        return results;
    }

}

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/SearchMessagesResponse.kt
package io.getstream.chat.android.client.api.models

import com.google.gson.annotations.SerializedName
import io.getstream.chat.android.client.models.SearchWarning

internal data class SearchMessagesResponse(
    val results: List<MessageResponse> = emptyList(),
    val next: String?,
    val previous: String?,
    @SerializedName("results_warning")
    val resultsWarning: SearchWarning?,
)
